src/json/load_config.c

```diff
@@ -359,6 +359,40 @@ static wolfsentry_errcode_t convert_sockaddr_address(JSON_TYPE type, const char
 
     if (sa->sa_family == WOLFSENTRY_AF_LINK) {
         int n = 0;
+#ifdef FREERTOS
+#pragma GCC diagnostic ignored "-Wformat"
+        /* FreeRTOS doesn't support %hhx, this alternative works but will
```

Comment:
This conversion will overrun the target pointers -- `%2x` is converting at-most-two-digit-long hex numbers to `int`, and for each of them, writing the whole int to the target pointer, but only one byte should be written.

To work around the lack of `%hhx`, you'll need to put 6 (1st case) or 8 (2nd case) scratch `int`s on the stack, and after the conversion, copy them to the corresponding bytes in `sa->addr`.

Once you've refactored thusly, you won't need to mask out `-Wformat` either.